backend/frontend/templates/frontend/course_component_details/page.html

```diff
@@ -1,33 +1,54 @@
 {% extends "frontend/base_logged_in.html" %}
 
 {% block content %}
-    <div class="mx-auto p-6 bg-white my-4 sm:w-11/12 lg:w-8/12">
+    <div class="mx-auto p-6 bg-white my-4 sm:w-11/12 lg:w-8/12 w-full overflow-auto">
         <h1 class="{{ styles.heading1 }}">
-            {{course_component.title}}
+            {{course_component.content_item.content_type_nice | title}}: {{ course_component.content_item.title }}
         </h1>
 
-        {% if course_component.submission_type_nice == "link"%}
-            <div class="grid gap-4 my-4">
-                <div class="p-2 border-2 rounded-md grid gap-2">
-                    <div class="bold p-2 border-2 rounded-md ">
-                        Tag names: 
-                        {% with course_component.tag_names|length as array_length %}
-                            {% for item in course_component.tag_names %}
-                                {{ item }}
-                                {% if not forloop.last %},{% endif %}
-                            {% endfor %}
-                        {% endwith %}
-                    </div>
-                
+        <div class="grid gap-4 my-4 w-full">
+            <div class="p-2 border-2 rounded-md grid gap-2">
+                <div class="p-2 border-2 rounded-md ">
+                    Tag names: 
+                    {% with course_component.content_item.tag_names|length as array_length %}
+                        {% for item in course_component.content_item.tag_names %}
+                            {{ item }}
+                            {% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    {% endwith %}
+                </div>
+            
+                <div class="p-2 border-2 rounded-md ">
+                    <p>
+                        Status : {{board_status}}
+                    </p>
+                </div>
+
+                {% if duration %}
```

Comment:
Why is this here?

Reply:
@sheenarbw, I was displaying duration information because of this point in the task spec:
![Screenshot 2024-04-07 124924](https://github.com/Umuzi-org/Tilde/assets/94048861/aa01a3fa-9626-472c-9b85-f1da0b0da6ca)